docs/DeveloperGuide.md

```diff
@@ -438,6 +445,8 @@ The following activity diagram will show how `RemarkCommand` can achieve the fun
 
 <img src="images/RemarkActivityDiagram.png"/>
 
+The remark feature has similar implementation as the [edit feature](#edit-feature), except that a `Remark` object is initialized instead of `EditCompanyDescriptor`.
```

Comment:
```suggestion
The remark feature has a similar implementation as the [edit feature](#edit-feature), except that a `Remark` object is initialized instead of `EditCompanyDescriptor`.
```